### ISSUE TITLE
Post Meta Text is not an accessible contrast ratio

### ISSUE THREAD

**Issue by kylek14** [aspirepress/site-theme#4]
The post meta information under the blog list entries does not meet current WCAG 2.1 AA contrast requirements. We should look at modifying this when we move blog posts to a separate page.

Screenshot of Issue:
![Screenshot from 2024-10-07 14-52-42](https://github.com/user-attachments/assets/78ec18e7-d95f-403a-8237-9e087acaec59)
